common/src/main/scala/org/apache/comet/CometConf.scala

```diff
@@ -605,6 +605,15 @@ object CometConf extends ShimCometConf {
       .booleanConf
       .createWithDefault(false)
 
+  val COMET_EXPR_ALLOW_INCOMPATIBLE: ConfigEntry[Boolean] =
+    conf("spark.comet.expression.allowIncompatible")
+      .doc(
+        "Comet is not currently fully compatible with Spark for all expressions. " +
+          "Set this config to true to allow them anyway. See compatibility guide " +
+          "for more information.")
```

Comment:
```suggestion
          "for more information. (docs/user-guide/compatibility.md)")
```

Reply:
I pushed a commit to update this by referencing a new `CometConf.COMPAT_GUIDE` variable with the link to the published documentation.

I regenerated the compatibility guide as well.